src/platform/export/exportDialog.ts

```diff
@@ -54,7 +59,8 @@ export class ExportDialog implements IExportDialog {
         const targetFileName =
             defaultFileName || !source
                 ? defaultFileName || ''
-                : `${path.basename(source.fsPath, path.extname(source.fsPath))}${extension}`;
+                : // eslint-disable-next-line local-rules/dont-use-fspath
```

Comment:
Do you need this linter exception, as you aren't using fspath.

Reply:
The linter doesn't allow us to read `source.fsPath` https://github.com/microsoft/vscode-jupyter/pull/10016/files#diff-905cbd240f7aea1a954bb235c3b83cb741403d43d5133c0cf32ac8da3a22644eR63